Formula/container-linux-config-transpiler.rb

```diff
@@ -0,0 +1,18 @@
+class ContainerLinuxConfigTranspiler < Formula
+  desc "Convert a Container Linux Config into Ignition"
+  url "https://github.com/coreos/container-linux-config-transpiler/archive/v0.5.0.tar.gz"
+  homepage "https://github.com/coreos/container-linux-config-transpiler"
+  sha256 "172e44796d39ec117584e121e73194cbcb56701407badc1899cea4607a343e15"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    system "make", "all"
+    bin.install "./bin/ct"
+  end
+
+  test do
+    system "#{bin}/ct", "--help"
```

Comment:
The test should be one that exercises and verifies the functionality of the software.

Reply:
A true test would involve transpiling a Linux Container configuration file or object, then validating that product with Ignition. That couldn't be done without incurring the cost of another dependency. It might be feasible (though brittle) to make a call to the CoreOS online validator (https://coreos.com/validate), but I don't expect the build framework to allow for that kind of behavior.